MTFO/ILRepack.targets

```diff
@@ -14,13 +14,13 @@
 
     <ILRepack
       Parallel="true"
+      Internalize="true"
```

Comment:
Note: `ILRepack.Lib.MSBuild` doesn't expose ILRepack's `RenameInternalized`. This means that exceptions (which are exempt from internalization?) can cause name resolution conflicts when importing MTFO in another project.